framework/console/controllers/FixtureController.php

```diff
@@ -431,11 +437,18 @@ private function getFixturesConfig($fixtures)
         $config = [];
 
         foreach ($fixtures as $fixture) {
-            $isNamespaced = (strpos($fixture, '\\') !== false);
-            $fullClassName = $isNamespaced ? $fixture . 'Fixture' : $this->namespace . '\\' . $fixture . 'Fixture';
+            // $isNamespaced = (strpos($fixture, '\\') !== false);
```

Comment:
Added one more commit - issue should be solved now.
Added fixture's relative path normalization to linux format in `findFixtures()` method;
reverted changes to `getFixturesConfig()` to its original (so `$isNamespaced` is calculated correctly) and added slash-replace from linux-path-backslashes to namespace slashes.